icevision/models/mmdet/models/detr/backbones/resnet_fpn.py

```diff
@@ -0,0 +1,20 @@
+__all__ = [
+    "r50_8x2_150e_coco",
+]
+
+from icevision.imports import *
+from icevision.models.mmdet.utils import *
+
+
+class MMDetRetinanetBackboneConfig(MMDetBackboneConfig):
```

Comment:
I think it's more appropriate to rename this class to `MMDetDetrBackboneConfig`

Reply:
Oh yes, I used the structure from retinanet and forgot to change the name